k8s/scripts/provision.sh

```diff
@@ -171,10 +171,10 @@ kind: MasterConfiguration
 apiServerExtraArgs:
   runtime-config: admissionregistration.k8s.io/v1alpha1
   ${admission_flag}: Initializers,NamespaceLifecycle,LimitRanger,ServiceAccount,DefaultStorageClass,DefaultTolerationSeconds,NodeRestriction,MutatingAdmissionWebhook,ValidatingAdmissionWebhook,ResourceQuota
-  feature-gates: "BlockVolume=true,CustomResourceSubresources=true,CSIBlockVolume=true"
+  feature-gates: "CustomResourceSubresources=true,CSIBlockVolume=true,VolumeSnapshotDataSource=true"
```

Comment:
this is for 1.11 and below versions, this feature gate doesn't exist in these versions.

Reply:
Reverted